.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy to Azure Web App
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+#
+# 1. Set up the following secrets in your repository:
+#   AZURE_WEBAPP_PUBLISH_PROFILE
+#
+# 2. Change these variables for your configuration:
+env:
+  AZURE_WEBAPP_NAME: fuget    # set this to your application's name
+  AZURE_WEBAPP_PACKAGE_PATH: '.'      # set this to the path to your web app project, defaults to the repository root
+  DOTNET_VERSION: '2.2.402'           # set this to the dot net version to use
```

Comment:
2.2 is out of support FYI

Reply:
Yeah my goal is to get up to .NET 5, but I don't want to deal with any code breaks right now. Or are you saying deployments will fail?